how-to/how-to-set-prometheus-and-grafana.md

```diff
@@ -0,0 +1,82 @@
+# How to setup Prometheus and `Grafana` as observability platform
+
+Prometheus and `Grafana` are open sources analytics & monitoring solution. This how-to will introduce a simple setup for one Linux node configuration, all Kata Containers, Prometheus and `Grafana` are running on the same host.
+
+> **Warning**: This how-to is only for evaluation purpose, you **SHOULD NOT** running it in production.
+
+* [Install Prometheus](#install-prometheus)
+* [Install `Grafana`](#install-grafana)
+* [Setup `Grafana`](#setup-grafana)
+  * [Create `datasource`](#create-datasource)
+  * [Import dashboard](#import-dashboard)
+
+Both Prometheus and `Grafana` can installed using pre-compiled distributions.
```

Comment:
What back ticks for Grafana but now Prometheus?

Also:
```suggestion
Both `Prometheus` and `Grafana` can be  installed using pre-compiled distributions.
```

Reply:
It's for the CI issues... ;-)

But now I have added Grafana to spelling check word list, so I removed all back ticks around Grafana.

Reply:
Thanks